Documentation/GettingStartedWithMRTKAndXRSDK.md

```diff
@@ -13,27 +13,64 @@ For MRTK 2.3, Windows Mixed Reality is supported on XR SDK.
 ### Required in Unity
 
 1. Go into Unity's Package Manager and install the Windows XR Plugin package, which adds support for Windows Mixed Reality on XR SDK. This will pull down a few dependency packages as well. Ensure the following all successfully installed:
-   1. XR Management
+   1. XR Plugin Management
    1. Windows XR Plugin
    1. XR Legacy Input Helpers
 1. Go to Edit > Project Settings.
-1. Click on the XR Plugin Management tab in the Project Settings window.
-1. Go to the Universal Windows Platform settings, then click the + to add a Windows MR Loader.
-1. Ensure that Initialize on Startup is checked.
-1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plugin Management and create a custom settings profile to change the defaults.
+1. Click on the XR Plug-in Management tab in the Project Settings window.
+1. Go to the Universal Windows Platform settings and ensure Windows Mixed Reality is checked under Plug-in Providers.
+1. Ensure that Initialize XR on Startup is checked.
+1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plug-in Management and create a custom settings profile to change the defaults. If the list of settings are already there, no profile needs to be created.
 
 ![Plugin management](Images/XRSDK/PluginManagement.png)
 
```

Comment:
It looks like the new screenshots were taken with the unity dark theme.  I think we decided that we would use the light theme for the docs moving forward.

Reply:
Oof yeah, good catch! I'll redo

Reply:
Updated